README.md

```diff
@@ -788,10 +788,11 @@ A pure text editor based on CodeMirror with syntax highlighting for HTML, CSS, J
 ### Properties
 
 | Name       | Type                       | Default        | Description                                                                                                                    |
-| ---------- | -------------------------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| ---------- | -------------------------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------ |
 | `project`  | `string PlaygroundProject` | `undefined`    | The project that this editor is associated with. Either the `<playground-project>` node itself, or its `id` in the host scope. |
 | `location` | `string`                   | `""`           |                                                                                                                                |
 | `htmlFile` | `string`                   | `"index.html"` | The HTML file used in the preview.                                                                                             |
+| `iframe`   | `HTMLIFrameElement         | null`          | `null`                                                                                                                         | A reference to the internal iframe element that is used to render the preview. |
```

Comment:
The `|` here is being interpreted as a column separator, so it broke the table. I think you can escape it like:

```
`HTMLIFrameElement \| null`
```